src/sentry/incidents/grouptype.py

```diff
@@ -79,7 +80,25 @@ class MetricAlertFire(GroupType):
     enable_escalation_detection = False
     detector_handler = MetricAlertDetectorHandler
     detector_validator = MetricAlertsDetectorValidator
-    detector_config_schema = {}  # TODO(colleen): update this
+    detector_config_schema = {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
+        "description": "A representation of a metric alert firing",
+        "type": "object",
+        "required": ["threshold_period", "detection_type"],
+        "properties": {
+            "threshold_period": {"type": "integer", "minimum": 1, "maximum": 20},
+            "comparison_delta": {
+                "type": ["integer", "null"],
+                "enum": [None, 300, 900, 6000, 86400, 604800, 2592000],
```

Comment:
This is based on the distinct values I could find in Redash, I need to go through and figure out if these are all intended and if so, make an enum somewhere to import.

Reply:
We have 4 rules that use a comparison delta of 6000 that were made using the API, going to add a restriction around this. The rest are valid input.